### PR TITLE
Food vision

### PR DESCRIPTION
**Pull request by mwitiderrick**

### CODE REVIEW

File: food-vision/readme.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Food vision with Layer
```
Comment (mecevit) [inline food-vision/readme.md:1]:
```suggestion
# Food vision with Layer
```

File: food-vision/readme.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+test_image = test_image / 255.0
+test_image = np.expand_dims(test_image, axis=0)
+prediction = image_model.predict(test_image)
+scores = tf.nn.softmax(prediction[0])
+scores = scores.numpy()
+base_dir = 'food-101/images'
+class_names = os.listdir(base_dir)
+f"{class_names[np.argmax(scores)]} with a { (100 * np.max(scores)).round(2) } percent confidence." 
+# > 'sashimi with a 1.3 percent confidence.'
+```
```
Comment (mecevit) [inline food-vision/readme.md:42]: Let's put a colab link here with the example code above

File: food-vision/readme.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+## Dataset
+The food dataset is obtained from http://data.vision.ee.ethz.ch/cvl/food-101.tar.gz
+
+## Model
+We train a convolution neural network model using TensorFlow to predict the type of food.
+
+https://app.layer.ai/layer/image-classification/models/food-vision https://app.layer.ai/layer/image-classification/models/food-vision#Sample-image https://app.layer.ai/layer/image-classification/models/food-vision?tab=logs&view=9efe73fc-bd13-4945-affd-bc1c96e6efd0#Metrics https://app.layer.ai/layer/image-classification/models/food-vision#Loss-plot https://app.layer.ai/layer/image-classification/models/food-vision#Accuracy-on-test-dataset https://app.layer.ai/layer/image-classification/models/food-vision#Accuracy-plot
```
Comment (mecevit) [inline food-vision/readme.md:49]: Any citation for the dataset?

**Review by mecevit — CHANGES_REQUESTED** [review]

**Review by mecevit — APPROVED** [review]
